easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.2.6-intel-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
```

Comment:
We have an easyblock for `netcdf4-python`, any reason you're not using that?

See https://github.com/hpcugent/easybuild-easyblocks/blob/master/easybuild/easyblocks/n/netcdf4_python.py